kofserver/scoreboard.py

```diff
@@ -57,9 +60,21 @@ def __init__(self):
     # )
     # In this case, Alice should get 30*5 points, Bob should get 30*5+30*10
     # points, and Eve doesn't get any point.
-    def LogTicks(gameName, playerName, portUptime, portScorePerSec, pidUptime, pidScorePerSec):
-        # TODO
-        pass
+    def LogTicks(self, gameName, playerName, portUptime, portScorePerSec, pidUptime, pidScorePerSec):
+        now = int(time.time())
+        sql = "INSERT INTO score (game_name, player_name, port_uptime, port_score_per_sec, pid_uptime, pid_score_per_sec, create_time) VALUES "
+        for i in range(len(playerName)):
+            sql += "('%s', '%s', %d, %d, %d, %d, %d)" % (gameName, playerName[i], portUptime[i], portScorePerSec[i], pidUptime[i], pidScorePerSec[i], now)
```

Comment:
Maybe use executemany for better performance and security (sql injection)?

Reply:
https://docs.python.org/2/library/sqlite3.html#sqlite3.Cursor.executemany

Reply:
Done